Library/Homebrew/keg_relocate.rb

```diff
@@ -81,7 +81,11 @@ def replace_text_in_files(relocation, files: nil)
       # when HOMEBREW_PREFIX == HOMEBREW_REPOSITORY we should use HOMEBREW_PREFIX for all relocations to avoid
       # being unable to differentiate between them.
       replacements[relocation.old_repository] = relocation.new_repository if HOMEBREW_PREFIX != HOMEBREW_REPOSITORY
-      changed = s.gsub!(Regexp.union(replacements.keys.sort_by(&:length).reverse), replacements)
+
+      replacement_regexes = replacements.keys.sort_by(&:length).reverse.map do |old_path|
+        /(?<![a-zA-Z0-9])#{old_path}/
```

Comment:
Wondering if there's any other characters we need to cover e.g. `._-` or similar? Maybe not.

Reply:
Yeah, I'm not sure. There are a whole bunch of things that are possible. It's probably just a game of "choose which are most-often used and, for any others that show up, either update this, use `inreplace`,  or use `pour_bottle?`.

- `./usr/local`: Probably not because relative paths are likely not actually referring to the prefix
- `_/usr/local`: Idk where this would show up so it's hard to say. It doesn't really seem like it would be referring to a path, though, so I'd say don't include unless it's a problem?
- `-/usr/local`: Again, not sure. Is it possible that it's trying to subtract the `/usr/local` prefix from another path? Might be worth including, then.

I'm really just not sure what kinds of things are used so unless anyone has anything specific they _know_ is being missed we may just want to move forward and handle new cases that come up?

Reply:
I'd love to collect data on this at some point but that's not trivial to do.

Reply:
The more I think about it: I think it's fine as-is.

> I'm really just not sure what kinds of things are used so unless anyone has anything specific they _know_ is being missed we may just want to move forward and handle new cases that come up?

Agreed!